terracumber_config/tf_files/SUSEManager-4.3-build-validation.tf

```diff
@@ -1050,13 +1076,32 @@ module "ubuntu2004-sshminion" {
   name               = "minssh-ubuntu2004"
   image              = "ubuntu2004o"
   provider_settings = {
-    mac                = "aa:b2:92:42:00:fa"
+    mac                = "aa:b2:92:42:00:ba"
```

Comment:
was this supposed to be sequential, and, therefore, `da`? if not, feel free to disregard my comment, but just wanted to double check

Reply:
same for the other `b`s down below, following this one

Reply:
you are perfectly correct, but it's already fixed, i have run a script to double-check.